### DISCUSSION TITLE
any plan support postgres?

### DISCUSSION THREAD

**Discussion by kaptinlin** [Infisical/infisical#659] [Feature Request]
mysql or postgres

**Comment by maidul98** [comment]
Yes, we are planning to start Postgres migration around end of September

**Reply by baek-sang** [reply]
not yet?

**Reply by dangtony98** [reply]
This is coming very soon ^

**Comment by smuskal** [comment]
Recommendations for a surgical replacement of a local PostgreSQL instead of Suprabase?  After a simple export from Suprabase and import locally, what next?

**Comment by maidul98** [comment]
Details for Postgress with Infisical will come out within the next 2 weeks

**Reply by compgeniuses** [reply]
2 weeks ago, its actually interesting i am coming about this now

**Reply by dangtony98** [reply]
This is coming out very soon; we're doing a lot of thorough testing which resulted in some delays — but for the better 😄

**Reply by DarkPhoenix2704** [reply]
Just saw the pg-migration scripts in the repo. Is the cloud running Postgres now?

**Reply by maidul98** [reply]
Yes it is. We'll have the guide on setting up postgres version for the community soon

**Reply by compgeniuses** [reply]
wonderful, looking forward to a postgres installation guide

**Comment by maidul98** [comment]
All methods are now postgres supported: https://infisical.com/docs/self-hosting/overview
If you were using the old Mongo version and would like to use Postgres based Infisical, please follow: https://infisical.com/docs/self-hosting/guides/mongo-to-postgres
